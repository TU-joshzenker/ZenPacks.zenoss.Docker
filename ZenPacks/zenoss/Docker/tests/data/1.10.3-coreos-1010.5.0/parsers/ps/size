CONTAINER ID                                                       IMAGE                    COMMAND                                          CREATED             STATUS                   PORTS                                                 NAMES               SIZE
22ae15131bc5091f9fe3e7f2a111b32a64cc7efa80cf7f4e5e0532e5b258d4d5   nginx                    "nginx -g 'daemon off;'"                         6 hours ago         Up 6 hours               443/tcp, 0.0.0.0:8080->80/tcp, 0.0.0.0:8081->81/tcp   nginx               2 B (virtual 182.7 MB)
1f109b4ae4c60446f095673ca370f414f569dc771587d72de8be200abcbeef46   dkuffner/docker-stress   "stress --cpu 8 --io 4 --vm 2 --vm-bytes 128M"   6 hours ago         Up 6 hours                                                                     pedantic_raman      0 B (virtual 369.9 MB)
4f09d7ef8eaac37345678857360a270277429568671c004f414e5a64482b4b20   hello-world              "/hello"                                         6 hours ago         Exited (0) 6 hours ago                                                         fervent_lovelace    0 B (virtual 967 B)
