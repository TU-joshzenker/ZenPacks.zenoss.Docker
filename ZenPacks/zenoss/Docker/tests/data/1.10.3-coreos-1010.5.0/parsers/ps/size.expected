# vim: ft=yaml
command: "/usr/bin/env sudo docker ps -a -s --no-trunc 2>/dev/null"

events:
  - device: "1.10.3-coreos-1010.5.0"
    component: "docker"
    summary: "received expected output from docker ps"
    severity: 0
    eventClassKey: "docker-ps-status"
    eventKey: "docker-ps-status"
    docker_command: "/usr/bin/env sudo docker ps -a -s --no-trunc 2>/dev/null"
    docker_output: |
      CONTAINER ID                                                       IMAGE                    COMMAND                                          CREATED             STATUS                   PORTS                                                 NAMES               SIZE
      22ae15131bc5091f9fe3e7f2a111b32a64cc7efa80cf7f4e5e0532e5b258d4d5   nginx                    "nginx -g 'daemon off;'"                         6 hours ago         Up 6 hours               443/tcp, 0.0.0.0:8080->80/tcp, 0.0.0.0:8081->81/tcp   nginx               2 B (virtual 182.7 MB)
      1f109b4ae4c60446f095673ca370f414f569dc771587d72de8be200abcbeef46   dkuffner/docker-stress   "stress --cpu 8 --io 4 --vm 2 --vm-bytes 128M"   6 hours ago         Up 6 hours                                                                     pedantic_raman      0 B (virtual 369.9 MB)
      4f09d7ef8eaac37345678857360a270277429568671c004f414e5a64482b4b20   hello-world              "/hello"                                         6 hours ago         Exited (0) 6 hours ago                                                         fervent_lovelace    0 B (virtual 967 B)

  - device: "1.10.3-coreos-1010.5.0"
    component: "22ae15131bc5091f9fe3e7f2a111b32a64cc7efa80cf7f4e5e0532e5b258d4d5"
    summary: "container status: up 6 hours"
    severity: 0
    eventClassKey: "dockerContainerStatus"
    eventKey: "dockerContainerStatus"

values:
  22ae15131bc5091f9fe3e7f2a111b32a64cc7efa80cf7f4e5e0532e5b258d4d5:
    size: 2
    size_virtual: 191574835
